### PR TITLE
lps-92427wchris

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:test:sf

File: modules/build.gradle [hunk 1]
```diff
@@ -22,13 +22,13 @@ ext {
...
+		"com.liferay.gradle.plugins.css.builder.version": "2.2.6",
 		"com.liferay.gradle.plugins.theme.builder.version": "2.0.6",
-		"com.liferay.gradle.plugins.version": "3.13.94",
-		"com.liferay.gradle.plugins.workspace.version": "1.10.18",
+		"com.liferay.gradle.plugins.version": "3.13.190",
+		"com.liferay.gradle.plugins.workspace.version": "2.0.2",
 		"com.liferay.portal.tools.bundle.support.version": "3.4.2",
-		"com.liferay.portal.tools.service.builder.version": "1.0.267",
+		"com.liferay.portal.tools.service.builder.version": "1.0.286",
 		"com.liferay.portal.tools.soy.builder.version": "3.0.4",
```
Comment (gamerson) [inline modules/build.gradle:32]: Do we need soy.builder anymore?

File: modules/build.gradle [hunk 2]
```diff
@@ -22,13 +22,13 @@ ext {
...
-		"com.liferay.css.builder.version": "2.1.2",
-		"com.liferay.gradle.plugins.css.builder.version": "2.2.4",
+		"com.liferay.css.builder.version": "2.1.3",
+		"com.liferay.gradle.plugins.css.builder.version": "2.2.6",
 		"com.liferay.gradle.plugins.theme.builder.version": "2.0.6",
-		"com.liferay.gradle.plugins.version": "3.13.94",
-		"com.liferay.gradle.plugins.workspace.version": "1.10.18",
+		"com.liferay.gradle.plugins.version": "3.13.190",
+		"com.liferay.gradle.plugins.workspace.version": "2.0.2",
 		"com.liferay.portal.tools.bundle.support.version": "3.4.2",
```
Comment (gamerson) [inline modules/build.gradle:30]: should this be bumped as well?

File: modules/build.gradle [hunk 3]
```diff
@@ -22,13 +22,13 @@ ext {
 	projectTemplateTokens = [
 		"biz.aQute.bnd.version": "3.5.0",
 		"com.liferay.ant.bnd.version": "2.0.52",
```
Comment (gamerson) [inline modules/build.gradle]: Should these be bumped?

File: modules/build.gradle [hunk 4]
```diff
@@ -22,13 +22,13 @@ ext {
 	projectTemplateTokens = [
 		"biz.aQute.bnd.version": "3.5.0",
```
Comment (gamerson) [inline modules/build.gradle:23]: I think we should bump the bnd to 4.2.0

File: modules/build.gradle (hunk 1 above)
Comment (lawrence-lee) [reply modules/build.gradle:32]: Yes. Form fields uses it

File: modules/build.gradle (hunk 2 above)
Comment (lawrence-lee) [reply modules/build.gradle:30]:
```
com.liferay.gradle.plugins.version": "3.13.190",
"com.liferay.gradle.plugins.workspace.version": "2.0.2",
"com.liferay.portal.tools.bundle.support.version": "3.4.2",
```
 these are already the latest

**Comment by gamerson** [comment]
ci:test:project-templates

File: modules/build.gradle (hunk 3 above)
Comment (lawrence-lee) [reply modules/build.gradle]: Peter wants me to remove this actually. He said Shuyang removed it from portal

File: modules/build.gradle (hunk 4 above)
Comment (lawrence-lee) [reply modules/build.gradle:23]: sure, I can do that. I can send a new PR tmrw when I remove `com.liferay.ant.bnd`

File: modules/build.gradle (hunk 1 above)
Comment (gamerson) [reply modules/build.gradle:32]: Got it

File: modules/build.gradle (hunk 2 above)
Comment (gamerson) [reply modules/build.gradle:30]: Very good.

File: modules/build.gradle (hunk 3 above)
Comment (gamerson) [reply modules/build.gradle]: Remove ant.bnd?  All of those plugins are still needed right?

**Comment by lawrence-lee** [comment]
ci:stop

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
pushed to peter https://github.com/petershin/liferay-portal/pull/502

**Pull request closed without merging**